2018 Robot/src/org/usfirst/frc/team4999/commands/TeleopNoPID.java

```diff
@@ -0,0 +1,51 @@
+package org.usfirst.frc.team4999.commands;
+
+import org.usfirst.frc.team4999.robot.Robot;
+import org.usfirst.frc.team4999.robot.choosers.ControlChooser;
+import org.usfirst.frc.team4999.robot.controllers.DriveController;
+import org.usfirst.frc.team4999.robot.subsystems.DriveSystem;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class TeleopNoPID extends Command {
+	
+	private DriveSystem drive = Robot.driveSystem;
+	private ControlChooser chooser = Robot.controlChooser;
+	
+	private boolean reversed;
+	
+    public TeleopNoPID() {
+        requires(drive);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+    	DriveController controller = chooser.getSelected();
+    	if(controller.getReverseDirection())
+    		reversed = !reversed;
```

Comment:
The DriveController should be calling the `getRawButtonPressed()` method on the controller it's wrapping. `getRawButtonPressed()` only returns true if the button was pressed since the last check. [Reference](http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/GenericHID.html)